citation-matching/citation-matching-core-code/src/main/java/pl/edu/icm/coansys/citations/DefaultInputReader.java

```diff
@@ -0,0 +1,60 @@
+package pl.edu.icm.coansys.citations;
+
+import org.apache.hadoop.io.BytesWritable;
+import org.apache.hadoop.io.Text;
+import org.apache.spark.api.java.JavaPairRDD;
+import org.apache.spark.api.java.JavaSparkContext;
+
+import pl.edu.icm.coansys.citations.data.MatchableEntity;
+import scala.Tuple2;
+
+/**
+ * Default reader for input documents and citations.
```

Comment:
Could add something more, e.g. when it is used and what are the assumptions related to the input data?